blazor/datagrid/filter-menu.md

```diff
@@ -223,4 +223,106 @@ The following screenshot represents CheckBox filter
 
 ### Limitations of using different filter types in different columns
 
-The different filter types such as Excel, Menu, and Checkbox can be defined in different columns of the same Grid. However, you cannot use these filter types along with filterBar type (default filter type). Because the filterbar type requires UI level changes with other filter types. For all other filter types, icons will be rendered in the column header.
\ No newline at end of file
+The different filter types such as Excel, Menu, and Checkbox can be defined in different columns of the same Grid. However, you cannot use these filter types along with filterBar type (default filter type). Because the filterbar type requires UI level changes with other filter types. For all other filter types, icons will be rendered in the column header.
+
+## Filtering using DateRangePicker
+
+By default, for the date column in the filter menu, filtering action is performed based on a single date value selected from the `SfDatePicker` component. The Grid also has an option to perform the filtering action between the range of date values by rendering the `SfDateRangePicker` component in the filter menu. This can be achieved by the filter template feature of the Grid.
+
+In the following sample, the `SfDateRangePicker` component is rendered in the filter template. Using the ValueChange event of the `SfDateRangePicker`, you can get the start and end date values which are needed to perform the filtering action. In the OnActionBegin event of the Grid, the default filter action is prevented by setting the args.Cancel as true. Then you can perform the filtering action by setting the custom predicate values.
```

Comment:
In the following sample, the `SfDateRangePicker` component is rendered in the filter template. Using the ValueChange event of the `SfDateRangePicker,` get the start and end date values needed to perform the filtering action. In the OnActionBegin event of the Grid, the default filter action is prevented by setting the args.Cancel as true. Then, perform the filtering action by setting the custom predicate values.